src/parameters.f90

```diff
@@ -243,7 +243,7 @@ subroutine parameter(input_i3d)
   endif
   
   !!==> Pasha
-  if(itype .eq. 14) then
+  if(itype .eq. 15) then
```

Comment:
It would really be better to have an `itype_XXX` for this case, however this is no worse than the current code :)